### PR TITLE
Cross mio updates for swm prawlers

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Added prawler info to node bulk load, eng bulk load, sensor bulk load.  They are technically WFPs as well so some of that terminology is the same, but I put Prawler where needed to differentiate from MMPs.  And then we realized we needed to differentiate the STC controller UID format from the WFP ones due to same sequential S/Ns being used (starting from -00001 for prawler STCs), so we decided on this format (in eng bulk load): CGCON-BSWSTC-xxxxx

Marine infrastructure Ss should help as reference as well (attached).  And table from WFP spec as well...

This was from Charlie @ McLane re prawler sensor SNs (and confirmed w/ Sheri's checks except for one CTD sensor where SN wasn't visible):
The serial numbers for the sensors are as follows:
16035-01 SBE s/n 709-0070  Aanderaa Optode s/n 4320
16035-02 SBE s/n 709-0068  Aanderaa Optode s/n 4321

[1100-00006_Marine_Infrastructure_Vocabulary_and_Reference_Designators_OOI_2024-03-20_Ver_2-05.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/14950146/1100-00006_Marine_Infrastructure_Vocabulary_and_Reference_Designators_OOI_2024-03-20_Ver_2-05.xlsx)
[3310-00003_Table 5.1.1.pdf](https://github.com/WHOIGit/ooicgsn-asset-management/files/14950173/3310-00003_Table.5.1.1.pdf)

### CODE REVIEW

File: bulk/eng_bulk_load-AssetRecord.csv
Comment (reedan88) [inline bulk/eng_bulk_load-AssetRecord.csv:1]: Looks good.

File: bulk/node_bulk_load-AssetRecord.csv
Comment (reedan88) [inline bulk/node_bulk_load-AssetRecord.csv:1]: Looks good.

File: bulk/sensor_bulk_load-AssetRecord.csv
Comment (reedan88) [inline bulk/sensor_bulk_load-AssetRecord.csv:1]: Looks good.

**Review by reedan88 — APPROVED** [review]

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 1]
```diff
@@ -1092,6 +1094,8 @@ CGINS-DOFSTK-03506,,Sensor,0,Dissolved Oxygen Fast Response: DOFST Series K,DOFS
 CGINS-DOFSTK-03523,,Sensor,0,Dissolved Oxygen Fast Response: DOFST Series K,DOFST for WFP Coastal ML14300-02,Sea-Bird Electronics,SBE 43F,43-3523,,20170627,,,Woods Hole Oceanographic Institution
 CGINS-DOFSTK-03535,,Sensor,0,Dissolved Oxygen Fast Response: DOFST Series K,DOFST for WFP Coastal ML14378-01,Sea-Bird Electronics,SBE 43F,43-3535,,20170907,,,Oregon State University
 CGINS-DOFSTK-04361,,Sensor,0,Dissolved Oxygen Fast Response: DOFST Series K,DOFST for WFP Coastal ML14378-01,Sea-Bird Electronics,SBE 43F,43-4361,,20230201,,,Oregon State University
+CGINS-DOFSTP-04320,,Sensor,0,Dissolved Oxygen Fast Response: DOFST Series P,DOFST for Prawler,Aanderaa,Optode 4831,4320,,,,,Woods Hole Oceanographic Institution
+CGINS-DOFSTP-04321,,Sensor,0,Dissolved Oxygen Fast Response: DOFST Series P,DOFST for Prawler,Aanderaa,Optode 4831,4321,,,,,Woods Hole Oceanographic Institution
```
Comment (cooleyky) [inline bulk/sensor_bulk_load-AssetRecord.csv:1098]: looks good

File: bulk/node_bulk_load-AssetRecord.csv [hunk 2]
```diff
@@ -67,6 +67,8 @@ CGVEH-CSPPCP-00005,N00125,Node,0,Coastal Surface Piercing Profiler Controller,"P
 CGVEH-CWFPCE-37801,,Node,0,Coastal Wire-Following Profiler,WFP COASTAL,McLane,MMP-7,ML14378-01,,20170907,,,Oregon State University
 CGVEH-CWFPCE-99103,A00285,Node,0,Coastal Wire-Following Profiler,WFP COASTAL,McLane,MMP-7,ML12991-03,,20130426,92259.49,210-3512,Oregon State University
 CGVEH-CWFPCE-99104,A00303,Node,0,Coastal Wire-Following Profiler,WFP COASTAL,McLane,MMP-7,ML12991-04,,20130510,92259.49,210-3512,Oregon State University
+CGVEH-CWFPCP-03501,,Node,0,Coastal Wire-Following Profiler,WFP COASTAL,McLane,Prawler,ML16035-01,,,,,Woods Hole Oceanographic Institution
+CGVEH-CWFPCP-03502,,Node,0,Coastal Wire-Following Profiler,WFP COASTAL,McLane,Prawler,ML16035-02,,,,,Woods Hole Oceanographic Institution
```
Comment (cooleyky) [inline bulk/node_bulk_load-AssetRecord.csv:71]: looks good

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 3]
```diff
@@ -167,6 +169,8 @@ CGCON-CSPPCP-00005,N00125.1,Sensor,0,Coastal Surface Piercing Profiler Controlle
 CGCON-CWFPCE-37801,,Sensor,0,Coastal Wire-Following Profiler Controller,WFP COASTAL Electronics #1,McLane,MMP-7,ML14378-01,,20170907,,,Oregon State University
 CGCON-CWFPCE-99103,A00285.1,Sensor,0,Coastal Wire-Following Profiler Controller,WFP COASTAL Electronics #1,McLane,MMP-7,ML12991-03,,20130426,0,,Oregon State University
 CGCON-CWFPCE-99104,A00303.1,Sensor,0,Coastal Wire-Following Profiler Controller,WFP COASTAL Electronics #1,McLane,MMP-7,ML12991-04,,20130510,0,,Oregon State University
+CGCON-CWFPCP-03501,,Sensor,0,Coastal Wire-Following Profiler Controller,Prawler Electronics,McLane,Prawler,ML16035-01,,,,,Woods Hole Oceanographic Institution
+CGCON-CWFPCP-03502,,Sensor,0,Coastal Wire-Following Profiler Controller,Prawler Electronics,McLane,Prawler,ML16035-02,,,,,Woods Hole Oceanographic Institution
```
Comment (cooleyky) [inline bulk/eng_bulk_load-AssetRecord.csv:173]: looks good

**Review by cooleyky — APPROVED** [review]
updates look good!

**Pull request merged**